templates/taxi/manufacturer_list.html

```diff
@@ -1,12 +1,15 @@
-{% extends "base.html" %}
+{% extends 'base.html' %}
+{% load crispy_forms_filters %}
 
 {% block content %}
   <h1>
     Manufacturer List
-    <a href="{% url 'taxi:manufacturer-create' %}" class="btn btn-primary link-to-page">
-      Create
-    </a>
+    <a href="{% url 'taxi:manufacturer-create' %}" class="btn btn-primary link-to-page">Create</a>
   </h1>
+  <form method="get" action="" class="form-inline">
+    <label><input type="text" name="search_query" class="form-control" placeholder="Search by name" value="{{ request.GET.search_query }}" /></label>
```

Comment:
The search input name here is `search_query`, but the `ManufacturerListView` expects `search`. Ensure consistency between the form and view to make the search functionality work.